data/settings.htm

```diff
@@ -390,7 +390,7 @@ <h5 class="card-header">
 						to Settings > Custom Stream. Your Stream key will be a string of letters and numbers
 						(not a URL).
 					</p>
-					<form action="/settings/update/" method="POST">
+					<form action="/settings/update/"  onsubmit="buttonDisable();">
```

Comment:
I think you accidentally deleted `method="POST"` here

Reply:
Gotit.